src/main/java/com/spotify/github/v3/checks/CheckRunOutput.java

```diff
@@ -87,4 +88,20 @@ public interface CheckRunOutput {
    * @return the optional
    */
   Optional<String> annotationsUrl();
+
+  /**
+   * Automatically validates the maximum length of properties.
+   * <p>
+   * GitHub does not validate these properly on their side (at least in GHE 3.2) and returns 422
+   * HTTP responses instead. To avoid that, let's validate the data in this client library.
+   */
+  @Value.Check
+  @SuppressWarnings("checkstyle:magicnumber")
+  default void check() {
+    // max values from https://docs.github.com/en/enterprise-server@3.5/rest/checks/runs#create-a-check-run
+    Preconditions.checkState(summary().map(String::length).orElse(0) <= 64 * 1024,
+        "'summary' exceeded max length of 65535 characters");
+    Preconditions.checkState(text().map(String::length).orElse(0) <= 64 * 1024,
+        "'text' exceeded max length of 65535 characters");
```

Comment:
```suggestion
    Preconditions.checkState(summary().map(String::length).orElse(0) < 64 * 1024,
        "'summary' exceeded max length of 65535 characters");
    Preconditions.checkState(text().map(String::length).orElse(0) < 64 * 1024,
        "'text' exceeded max length of 65535 characters");
```

Reply:
64*1024 = 65536
Maximum length allowed is 65535 characters

Reply:
Already resolved, thanks